### ISSUE TITLE
CTRL+Enter for submitting comment

### ISSUE THREAD

**Issue by kytrinyx** [exercism/exercism#3952]
_Imported from https://github.com/exercism/-DEPRECATED-PLEASE-USE-exercism.io-REPO-v2-feedback/issues/94._
_Originally posted by @NobbZ._

#

Just hitting CTRL and Enter in conjunction is a way to submit comments that I'm used to from many forums, github and even exercism V1 ;) It would be nice to have it back in V2.

**Comment by ErikSchierboom** [comment]
Small bump: GitHub has just released this: https://github.com/github/hotkey. It looks like a super nice library to add keyboard shortcuts.

**Comment by iHiD** [comment]
Once https://github.com/exercism/website/pull/526 is merged, I'll accept a PR for this that has no side-effects other than CTRL-Enter for form submission. It'll need to be consistent everywhere where there are comments (blogs, solutions, mentoring, etc) and DRY.

**Comment by iHiD** [comment]
Closing as part of https://github.com/exercism/exercism/issues/2053.
